### DISCUSSION TITLE
비회원의 지원 내역 조회

### DISCUSSION THREAD

**Discussion by gayeong0916** [team-final/albaform#112] [General]
![화면 캡처 2024-09-25 222612](https://github.com/user-attachments/assets/96f206df-26b8-4d40-aeaa-d963628701f7)
비회원일 때 : 지원하기 버튼을 누를 수 있게 할까 말까?

**Comment by JuhyeokC** [comment]
![image](https://github.com/user-attachments/assets/56265413-c6c0-4db9-b03e-e520c2edcff3)

1. 기본 요구 사항에서는 비회원이 지원, 내역 확인을 할 수 있게 되어있음.

2. 초기 회의 때 비회원은 로그인(회원가입)으로 유도하기로 함.

**Comment by gayeong0916** [comment]
비회원일 때 -> 내 지원 내역 보기 모달창 = 부가 기능이라고 생각

일단, 지금은 '회원'일 경우만 생각하기
